themes/bootstrap3/js/lightbox.js

```diff
@@ -416,7 +416,7 @@ $(document).ready(function() {
   $('#modal').on('hidden.bs.modal', Lightbox.closeActions);
   /**
    * If a link with the class .modal-link triggers the lightbox,
-   * look for a title="" to use as our lightbox title.
+   * look for a title="&nbsp;" to use as our lightbox title.

```

Comment:
I don't think the &nbsp; belongs here... maybe it would be better to rephrase this comment to say "look for a title attribute to use as our lightbox title."